### PR TITLE
Release/0.3.0

### PR DESCRIPTION
**Pull request by dazza-codes**
This release backs out threading for triannon POSTS in the DMS ETL, only because it fails to save the anno URIs in the tracking log file.  This is required for repeating the ETL.  There may be alternative solutions for the tracking problem with threading, but that will be left for another release.

### CODE REVIEW

**Comment by dazza-codes** [comment]
@ndushay and/or @azaroth42 - this is a relatively minor change to the DMS ETL.  If you have time, take a look, although I may do the merge to keep moving with cutting a new gem and working on the DMS ETL replay.

File: lib/annotations2triannon/open_annotation.rb [hunk 1]
```diff
@@ -27,10 +27,7 @@ def initialize(graph=RDF::Graph.new, id=nil)
       else
         @graph = graph
         raise ArgumentError, 'graph must be an open annotation' unless is_annotation?
-        if id.nil?
-          @id = get_id
-        else
-        end
+        id.nil? ? @id = get_id : @id = id
```
Comment (ndushay) [inline lib/annotations2triannon/open_annotation.rb:30]: this is fine.  i'm wondering if this reads better:

@id = id.nil? ? get_id : id

**Comment by ndushay** [comment]
:+1:

File: lib/annotations2triannon/open_annotation.rb (hunk 1 above)
Comment (dazza-codes) [inline lib/annotations2triannon/open_annotation.rb:30]: Yes!  I know that pattern and I was kicking myself about this!  That's what you get from doing some vim line manipulations without starting from scratch.  (Will fix this in the next release.)

**Pull request merged**